### PR TITLE
add approval ui dimensions

### PR DESCRIPTION
**Pull request by douglaseggleton**
Adds popup dimension configuration or fallback to standard size.

## Release notes

Adds the capability for popup redirects to have a custom size set by individual payment processors.

### CODE REVIEW

File: packages/embed/webpack.dev.js [hunk 1]
```diff
@@ -49,5 +50,8 @@ module.exports = {
     new HtmlWebpackPlugin({
       title: 'Gr4vy - Embed',
     }),
+    new DefinePlugin({
```
[the remark this answers is not in the document]
Comment (douglaseggleton) [reply packages/embed/webpack.dev.js:53]: I noticed it was broken on the development build.

File: packages/embed/src/popup/popup.ts [hunk 2]
```diff
@@ -3,14 +3,20 @@ import { mutableRef } from '../utils'
...
 export const createPopupController = (
   popup = mutableRef<{ popup: Window; stopCallback: () => void }>(),
   subject: SubjectManager
 ) => {
   subject.approvalStarted$.subscribe(() => {
     if (subject.mode$.value()?.popup) {
       popup.current = openPopup(
-        popupFeatures(500, 589),
+        popupFeatures(
+          subject.mode$.value().popup.width || DEFAULT_POPUP_WIDTH,
```
[the remark this answers is not in the document]
Comment (douglaseggleton) [reply packages/embed/src/popup/popup.ts]: It should not be undefined as its inside an if condition - but I have just refactored the logic to make it clearer.

**Review by luca-gr4vy — APPROVED** [review]
looks good to me 👍

**Comment by gr4vy-code** [comment]
:rocket: PR was released in [`v2.11.6`](https://github.com/gr4vy/gr4vy-embed/releases/tag/v2.11.6) :rocket:

**Pull request merged**